taskcluster/scripts/pipeline/train-taskcluster.sh

```diff
@@ -62,6 +63,7 @@ case "$pretrained_model_mode" in
         "$best_model_metric" \
         "$alignments" \
         "$seed" \
+        "$teacher_mode" \
```

Comment:
Question: Do you know if we have any tests for training continuation? It would be nice to have coverage here. (no action needed in this PR)

Reply:
We don't have any. It's tricky to test. But it has the same entry point as regular training. It might be easier to test with your recent refactoring. I found only a test that checks that taskcluster-train.py propagates arguments.